app/views/user_mailer/review_notification.text.erb

```diff
@@ -0,0 +1,10 @@
+You have a review, <%= @user.email %>
+
+Your program <%= @program.title %> has been
+reviewed by <%= @reviewer.email%>.
+Click <%= link_to "here", @program.url %> to view.
+
+
+                        As always,
+                        fuck off,
+                        Staff of couch-potatoes
```

Comment:
**Note**: Employers will read your code! Don't put anything in here that would keep you from getting a job!